### DISCUSSION TITLE
Como faço Pegar retorno de Stream

### DISCUSSION THREAD

**Discussion by ebnezerjrs** [viniciussanchez/dataset-serialize#208] [General]
bom dia, estou tentando pegar o retorno desse exemplo de Stream abaixo no delphi com Lresponse nao consigo

no postman consigo ve o retorno.

![image](https://github.com/viniciussanchez/dataset-serialize/assets/95051045/f3c2e1cc-7ea0-4749-8ef9-25e8175db8bd)

**Comment by viniciussanchez** [comment]
te respondi no tópico que voce criou no RESTRequest4Delphi
